res/values-pt-rBR/strings.xml

```diff
@@ -289,7 +291,7 @@
     <string name="No_such_user_found_">"Nenhum usuário encontrado."</string>
     <string name="No_open_public_clans_exist_">"Não existem clãs públicos abertos."</string>
     <string name="You_aren_t_invited_to_that_clan_">"Você não está convidado para este clã."</string>
-    <string name="No_such_clan_exists_">"Não existe tal clã."</string>
+    <string name="No_such_clan_exists_">"Este clã não existe."</string>
```

Comment:
Since "such" is visible in that string, I've used the current phrase:

```suggestion
    <string name="No_such_clan_exists_">"O clã ao qual se destina não existe."</string>